bioblend/galaxy/histories/__init__.py

```diff
@@ -37,6 +38,16 @@ def create_history(self, name=None):
             payload['name'] = name
         return self._post(payload)
 
+    def import_history(self, file_path=None, url=None, files_attached=False):
+        if file_path:
+            files_attached=True
+            archive_file=attach_file(file_path)
+            payload = dict(archive_source='', archive_file=archive_file)
```

Comment:
```suggestion
            payload = dict(archive_source='', archive_file=archive_file, archive_type="file")
```
The archive_type is passed on to the `imp_history_from_archive` tool